README.md

```diff
@@ -140,6 +140,22 @@ func application(_ application: UIApplication, didFailToRegisterForRemoteNotific
 }
 ```
 
+### Location pushes
+
+Starting with iOS 15, Apple supports power efficient way to request location trough [location push service extension](https://developer.apple.com/documentation/corelocation/creating_a_location_push_service_extension).
+
+To use this new functionality you should obtain Apple entitlments, create location push service extension and make some tweaks in your push notifications code within your app:
+
+* Add Apple location pushes entitlments into your project ("AblyLocationPush" folder).
+
+* In addition to saving your regular device token by calling `ARTPush.didRegisterForRemoteNotifications(withDeviceToken:realtime:)` you should request another device token specifically for the location pushes purpose. You do so by calling `CLLocationManager.startMonitoringLocationPushes(completion:)` within a `ARTPushRegistererDelegate.didActivateAblyPush(:)` delegate callback or whenever is appropriate in your app.
+
+* Once you receive the location push token, save it by calling a new `ARTPush.didRegisterForLocationNotifications(withDeviceToken:realtime:)` method (note the "Location" word in the name of this method).
```

Comment:
And then what? How do I know when it's been successfully submitted? We need to document this and I also think that the example app should demonstrate it.

Reply:
https://github.com/ably/ably-cocoa/pull/1771/commits/794e634e187b3df8f0a56ffac1243e41e592211e